File: Scripts/newTask/file-input-output/swift/file-input-output.swift
import Foundation

for i in 0...1000000 {

let file: FileHandle? = FileHandle(forReadingAtPath: "../input.txt")

if file != nil {
    // Read all the data
    let data = file?.readDataToEndOfFile()

    // Close the file
    file?.closeFile()

    // Convert our data to string
    let text = String(data: data!, encoding: String.Encoding.utf8)
    let contains = text!
    let path = "output.txt"
	do {
    // Write contents to file 
   
    try contains.write(toFile: path, atomically: false, encoding: String.Encoding.utf8)
}
catch let error as NSError {
    print("Ooops! Something went wrong: \(error)")
}
}
else {
    print("Ooops! Something went wrong!")
}

}

File: Scripts/newTask/file-input-output/java/FileInputOutput.java
import java.io.*;
public class FileInputOutput {
  public static void main (String[] args) throws IOException {

    for ( int i = 0; i < 1000000; ++i) {

    BufferedReader br = new BufferedReader(new FileReader("../input.txt"));
    BufferedWriter bw = new BufferedWriter(new FileWriter("output.txt"));
    String line;
    while ((line = br.readLine()) != null) {
      bw.write(line);
      bw.newLine();
    }
    br.close();
    bw.close();
 }
  }
}
